packages/core/src/components/forms/numericStepper.tsx

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+
+import * as Classes from "../../common/classes";
+import * as Keys from "../../common/keys";
+import { removeNonHTMLProps } from "../../common/props";
+import { HTMLInputProps, IControlledProps, IIntentProps, IProps } from "../../common/props";
+
+import { Button } from "../button/buttons";
+import { InputGroup } from "./inputGroup";
+
+export type ButtonPosition = "none" | "left" | "right" | "split";
+
+export interface INumericStepperProps extends IControlledProps, IIntentProps, IProps {
+
+    /**
+     * The button configuration with respect to the input field.
+     * @default right
+     */
+    buttonPosition?: ButtonPosition;
```

Comment:
Haven't done much research yet but want to get your take: should we be opinionated about this and choose a good default, or should we leave it configurable?